### PR TITLE
guide: Managing Experiments

### PR DESCRIPTION
**Pull request by jorgeorpinel**
Extracted from https://github.com/iterative/dvc.org/pull/2690#pullrequestreview-735446663

- [x] Un-nest from #2690 when that PR is merged
- [ ] Combine and summarize the motivation text in the section index
- [ ] Finish Managing Exps guide (mainly about `exp show/compare`)
- [ ] Remove deep details from the command reference that can be moved here instead

Maybe also include (if natural)
- [ ] https://github.com/iterative/dvc.org/issues/2736
- [ ] https://github.com/iterative/dvc.org/issues/2737

### CODE REVIEW

File: content/docs/user-guide/experiment-management/managing-experiments.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+Experiments created with `dvc exp run` will have an auto-generated name like
+`exp-bfe64` by default. It can be customized using the `--name` (`-n`) option:
+
+```dvc
+$ dvc exp run --name cnn-512 --set-param model.conv_units=512
+```
+
+When you create an experiment, DVC generates a Git-like SHA-1 hash from its
+contents. This is shown when you [queue experiments] with `--queue`:
```
Comment (jorgeorpinel) [inline content/docs/user-guide/experiment-management/managing-experiments.md:18]: > From https://github.com/iterative/dvc.org/pull/2690#discussion_r694263173

> This is actually different from the hash generated after the experiments is committed, so I'm not sure it makes sense to mention it here.

File: content/docs/user-guide/experiment-management/managing-experiments.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+[queue experiments]:
+  /doc/user-guide/experiment-management/running-experiments#the-experiments-queue
+
+```dvc
+$ dvc exp run --queue -S model.conv_units=32
+Queued experiment '6518f17' for future execution.
+```
+
+After running queued experiment, DVC uses the regular name mentioned earlier.
```
Comment (jorgeorpinel) [inline content/docs/user-guide/experiment-management/managing-experiments.md:28]: > From https://github.com/iterative/dvc.org/pull/2690#discussion_r694264184

> The experiment name should show in dvc exp show even for queued experiments now.

File: content/docs/user-guide/experiment-management/index.md [hunk 3]
```diff
@@ -1,5 +1,18 @@
 # Experiment Management
```
Comment (jorgeorpinel) [inline content/docs/user-guide/experiment-management/index.md:1]: > From https://github.com/iterative/dvc.org/pull/2690#discussion_r694266392

> I guess I'm the reason for this new section? I'm okay with leaving this in Running experiments depending on what others think. It does seem a bit disjointed as is. I think we need to at least link here from Running experiments so it's clear how to see and compare experiments.

**Comment by iesahin** [comment]
For 

> Finish Managing Exps guide (mainly about exp show/compare)

I have #2737. 

I think I can also create issues for other items as well.

**Comment by jorgeorpinel** [comment]
> I have #2737.

Yeah @iesahin I created this one based on https://github.com/iterative/dvc.org/pull/2690#pullrequestreview-735446663 before noticing your Exp Compare guide.

I think it could be included in this guide for now, as comparing them is probably a basic need to manage them. May not even need an explicit section but rather implied throughout this guide.

**Comment by jorgeorpinel** [comment]
@iesahin should we close this and turn it into an issue?

**Comment by jorgeorpinel** [comment]
Thanks @iesahin !

**Pull request closed without merging**